mlflow/server/spans_router.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from fastapi import APIRouter
+from pydantic import BaseModel
+
+from mlflow.server.handlers import STATIC_PREFIX_ENV_VAR
+
+prefix = os.environ.get(STATIC_PREFIX_ENV_VAR, "")
+router = APIRouter(prefix=f"{prefix}/api/2.0")
```

Comment:
```suggestion
router = APIRouter(prefix=f"{prefix}/api/3.0")
```

Let's start using 3.0 APIs, as of MLflow 3